src/components/lend/LendMarketCard.tsx

```diff
@@ -76,6 +78,17 @@ export const LendMarketCard: React.FC<LendMarketCardProps> = ({
             <Typography variant="body1">
               {formatter.toPercentage(reserve.estimates.supplyApy)}
             </Typography>
+            {reserve.supplyEmissions && (
+              <FlameIcon
+                width={22}
+                height={22}
+                title={` This asset earns ${getEmissionsPerDayPerUnit(
+                  reserve.supplyEmissions?.config.eps || BigInt(0),
+                  reserve.estimates.supplied,
+                  reserve.config.decimals
+                )} BLND/day emissions`}
```

Comment:
can reformat this string slightly in all locations?

`This asset earns ### BLND a day per ${reserve.tokenMetadata.symbol}`